presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -152,9 +152,9 @@
 
     private static final String UNSUPPORTED_CORRELATED_SUBQUERY_ERROR_MSG = "line .*: Given correlated subquery is not supported";
 
-    protected AbstractTestQueries(QueryRunnerSupplier supplier)
+    protected AbstractTestQueries(QueryRunnerSupplier supplier, FeatureSelection featureSelection)
```

Comment:
add constructor
```
AbstractTestQueries(QueryRunnerSupplier supplier) {
    this(supplier, FeatureSelection.all().build());
}
```
instead of modifying all test constructors.